### ISSUE TITLE
Use Pillow to handle images

### ISSUE THREAD

**Issue by anuschka** [CodeCatz/TrackCat#220]
As in this tutorial: 
>http://pillow.readthedocs.org/en/latest/handbook/tutorial.html

**Comment by ialja** [comment]
Btw, Pillow is already in use.

**Comment by anuschka** [comment]
You are totally right @ialja  :+1: but we could perhaps think about :thought_balloon:  (next release) using some of its image manipulation features in TrackCat.
